### PR TITLE
Added numbers colorize by filter

### PR DESCRIPTION
**Pull request by Tal-D**

### CODE REVIEW

File: spec/contact_fixer_spec.rb [hunk 1]
```diff
@@ -4,9 +4,39 @@
 
 describe ContactFixer do
   before(:each) do
+    @contact_phone_number = "976-shoe"
+    @phone_numbers_filter = "976"
+    @highlighted_phone_number = @phone_numbers_filter.green + "-shoe"
```
[the remark this answers is not in the document]
Comment (Tal-D) [reply spec/contact_fixer_spec.rb:9]: Done

File: lib/contact_fixer.rb [hunk 2]
```diff
@@ -19,7 +21,16 @@ def get_fixed_display_name(display_name)
     end
   end
 
-  def print_connection(person)
+  def print_connection_phone_numbers(phone_numbers, raw_filter)
+    if raw_filter.nil?
+      @output.puts "- " + phone_numbers.map { |phone_number| phone_number.value }.inspect
```
[the remark this answers is not in the document]
Comment (Tal-D) [reply lib/contact_fixer.rb:26]: Done

File: spec/contact_fixer_spec.rb [hunk 3]
```diff
@@ -4,9 +4,39 @@
...
     @out = StringIO.new
   end
 
+  describe '.print_connection_phone_numbers' do
+    before(:each) do
+      @mock_phone_number = instance_double("phoneNumbers")
```
[the remark this answers is not in the document]
Comment (Tal-D) [reply spec/contact_fixer_spec.rb:15]: I assumed that you referred to the @mock_phone_number variable.

[the remark this answers is not in the document]
Comment (georgettica) [reply spec/contact_fixer_spec.rb]: yes

File: lib/contact_fixer.rb [hunk 4]
```diff
@@ -2,10 +2,19 @@
 
 class ContactFixer
   def initialize(contacts_api, output)
```
Comment (georgettica) [inline lib/contact_fixer.rb:4]: why not allow the filter to be set here aswell?

File: lib/contact_fixer.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (georgettica) [reply lib/contact_fixer.rb:35]: that's not what I meant, it was more for the `@output.puts "- " + phone_numbers.map { |phone_number| phone_number.value }.inspect` section

File: lib/contact_fixer.rb (hunk 4 above)
Comment (Tal-D) [reply lib/contact_fixer.rb:4]: Done

File: lib/contact_fixer.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (Tal-D) [reply lib/contact_fixer.rb]: Updated

**Review by georgettica — APPROVED** [review]

**Pull request merged**